src/main/java/io/temporal/worker/WorkerOptions.java

```diff
@@ -301,15 +304,15 @@ public int hashCode() {
   public String toString() {
     return "WorkerOptions{"
         + "maxActivitiesPerSecond="
```

Comment:
[Nit] let's update the strings for these two variables as well

Reply:
done